test/ha/resilience_test.go

```diff
@@ -0,0 +1,169 @@
+//go:build e2e
```

Comment:
nit: resilience_test.go is a bit generic - can we name this file `workload_test.go`

Reply:
renamed the file